src/main/java/com/zufar/onlinestore/payment/controller/PaymentController.java

```diff
@@ -52,4 +64,23 @@ public ResponseEntity<PaymentDetailsDto> getPaymentDetails(@PathVariable Long pa
         return ResponseEntity.ok()
                 .body(retrievedPayment);
     }
+
+    /**
+     * This endpoint is used only until we have an implementation of this logic on the frontend side.
+     * It will come in handy for testing the API.
+     */
+    @PostMapping("/method")
+    public ResponseEntity<String> getPaymentMethodId(@RequestBody CreatePaymentMethodDto paymentRequest) throws StripeException {
```

Comment:
why it is written `getPaymentMethodId` but ` @PostMapping("/method")` and not  `@PostMapping("/methodId")`?

Reply:
> why it is written `getPaymentMethodId` but ` @PostMapping("/method")` and not `@PostMapping("/methodId")`?

Done